frontend/webapp/utils/__init__.py

```diff
@@ -16,3 +18,100 @@ def flatten(items, seqtypes=(list, tuple)):
         else:
             res.append(item)
     return res
+
+
+def calculate_more_specifics(prefix, min_length, max_length):
+    prefix_list = []
+    for prefix_length in range(min_length, max_length + 1):
+        prefix_list.extend(prefix.subnets(new_prefix=prefix_length))
+    return prefix_list
+
+
+def translate_rfc2622(input_prefix, just_match=False):
```

Comment:
@slowr do you think I should keep this piece of code? It is not actually used in the frontend, but might be useful if we need frontend-side translation. I can also remove it altogether for now. Let me know your thoughts on this. Same for the function above.

Reply:
we can keep it commented out

Reply:
ok will do